rqt_joint_trajectory_controller/rqt_joint_trajectory_controller/joint_trajectory_controller.py

```diff
@@ -461,12 +461,9 @@ def _joint_widgets(self):  # TODO: Cache instead of compute every time?
 
 
 def _jtc_joint_names(jtc_info):
-    # NOTE: We assume that there is at least one hardware interface that
-    # claims resources (there should be), and the resource list is fetched
-    # from the first available interface
 
     joint_names = []
-    for interface in jtc_info.claimed_interfaces:
+    for interface in jtc_info.required_state_interfaces:
```

Comment:
Do we need this changes?

Reply:
thb, I don't remember well (this PR is 18months old). But according to the note in the initial commit, I believe yes :D

Reply:
yes. the thing is, that you **can** send a trajectory messages including the non-command DOF now. (does not make much sense for the GUI maybe, but you will see the effect then).
should not change anything for existing configs, right?

Reply:
Thanks for the explanation